Description: Disable autorescan, this is done by the udev rule.
Author: Colin Watson <cjwatson@ubuntu.com>

=== renamed file 'pbbuttonsd.conf' => 'pbbuttonsd.conf'
Index: pbbuttonsd-0.8.1/pbbuttonsd.cnf.in
===================================================================
--- pbbuttonsd-0.8.1.orig/pbbuttonsd.cnf.in
+++ pbbuttonsd-0.8.1/pbbuttonsd.cnf.in
@@ -6,7 +6,7 @@
 [SYSTEM]
 #userallowed=paranoid
 CmdTimeout=8
-autorescan=true
+autorescan=false
 
 [MODULE DISPLAY]
 #LCD_Brightness=90
